cypress/e2e/awx/resources/hosts.cy.ts

```diff
@@ -29,79 +29,33 @@ describe('host and inventory host', () => {
   });
 
   //tests
-  it.skip('can create, edit and delete a inventory host', () => {
+  it('can create, edit and delete a inventory host', () => {
```

Comment:
Is this scaffolding from Sarah? I though the inventory hosts tests will be in inventory file?

Reply:
no
as you can see in line 38 it's not only inventory hosts